packages/run-protocol/test/test-gov-collateral.js

```diff
@@ -586,4 +586,4 @@ test('Committee can raise debt limit', async t => {
   });
 });
 
-// test.todo('users can open vaults');
+test.todo('users can open vaults');
```

Comment:
It's a cast so that `t.context` is statically typed. E.g. in this file `t.context.zoe` is known to be `ZoeService`.

<img width="490" alt="Screen Shot 2022-07-17 at 8 10 33 AM" src="https://user-images.githubusercontent.com/21505/179404601-98be1d94-08a7-4c79-af0b-cc561207f716.png">
.

Reply:
Wow, that's good to know. Resolving.